app/javascript/schools/courses/components/students_editor/StudentsEditor__SearchableTagList.res

```diff
@@ -55,29 +55,33 @@ let search = (state, send, allowNewTags, selectedTags, unselectedTags, addTagCB)
 
 let reducer = (_state, searchString) => searchString
 
+let showTags = (tags, send, removeTagCB) => {
+  {tags
```

Comment:
Wasn't this code auto-formatted with refmt? These braces are superfluous, and would be removed by refmt. 🤔

Reply:
fixed